src/main/java/com/napier/sem/DAO.java

```diff
@@ -53,20 +52,65 @@ public DAO(Connection connection)
     /**
      * Constructs the SQL query required and returns the result of the
      * query.
+     *
+     * @return An ordered list of countries in the world sorted by descending population
+     */
+    public ArrayList<Country> allCountries() {
+        // Define the SQL query as a string
+        String statementString = "SELECT code, name, continent, region, population, (\n" +
+                "    SELECT name\n" +
+                "    FROM city ci\n" +
+                "    WHERE countrycode = co.code\n" +
+                "        AND ci.id = co.capital\n" +
+                "    ) AS capital\n" +
+                "FROM country co\n" +
+                "ORDER BY population DESC\n";
+//              "LIMIT 20";
```

Comment:
i think this comment probably shouldn't be here, it's probably useful for testing but i think we should take it out before the code review. the \n should be taken out of line 67 too.